CHANGELOG.md

```diff
@@ -54,127 +55,130 @@ For more details about the updated installation, please visit the official docs:
+- :fireworks: **Newly open-sourced plugin**: the HTTP [proxy-cache
+  plugin](https://github.com/kong/kong-plugin-proxy-cache) (previously only
+  available in Enterprise) is now bundled in Kong.
   [#4650](https://github.com/Kong/kong/pull/4650)
-- :fireworks: The `request-transformer` and `request-transformer-advanced`
-  (previously only available in Enterprise) plugins were merged
+- :fireworks: **Newly open-sourced plugin capabilities**: The
+  [request-transformer
+  plugin](https://github.com/Kong/kong-plugin-request-transformer) now includes
+  capabilities previously only available in Enterprise, among which templating
+  and variables interpolation.
   [#4658](https://github.com/Kong/kong/pull/4658)
 - Logging plugins: log request TLS version, cipher, and verification status.
   [#4581](https://github.com/Kong/kong/pull/4581)
   [#4626](https://github.com/Kong/kong/pull/4626)
-- Plugin development: inheriting from `BasePlugin` is now optional
+- Plugin development: inheriting from `BasePlugin` is now optional. Avoiding
+  the inheritance paradigm improves plugins' performance.
   [#4590](https://github.com/Kong/kong/pull/4590)
 
 ### Fixes
 
-#### Core
+##### Core
 
-- Active healthchecks: `http` checks aren't performed for TCP/TLS services anymore;
-  only `tcp` healthchecks are performed.
+- Active healthchecks: `http` checks are not performed for `tcp` and `tls`
+  Services anymore; only `tcp` healthchecks are performed against such
+  Services.
   [#4616](https://github.com/Kong/kong/pull/4616)
 - Fix an issue where updates in migrations would not correctly populate default
   values.
   [#4635](https://github.com/Kong/kong/pull/4635)
-- Improvements in reentrancy of Cassandra migrations.
+- Improvements in the reentrancy of Cassandra migrations.
   [#4611](https://github.com/Kong/kong/pull/4611)
-- Fix to an issue introduced in `1.2.0rc1` where `init_worker` was not being
-  called on plugins that implemented it
-  [#4670](https://github.com/Kong/kong/pull/4670)
-- Fix to an issue where `:new` was not always called when Plugins
-  deep-inherited from BasePlugin
-  [#4671](https://github.com/Kong/kong/pull/4671)
-- Fix an issue where Postgres would not bootstrap the schema with an account
-  with limited permissions. [#4506](https://github.com/Kong/kong/pull/4506)
+- Fix an issue causing the PostgreSQL strategy to not bootstrap the schema when
+  using a PostgreSQL account with limited permissions.
+  [#4506](https://github.com/Kong/kong/pull/4506)
 
+##### CLI
 
-#### CLI
-
-- `kong migrations [up|finish] -f` commands do not run if there are no executed
-  migrations.
-  [#4617](https://github.com/Kong/kong/pull/4617)
-- Fix db_import to upsert via deterministic v5 UUIDs
+- Fix `kong db_import` to support inserting entities without specifying a UUID
+  for their primary key. Entities with a unique identifier (e.g. `name` for
+  Services) can have their primary key omitted.
```

Comment:
This was specified as a bug fix, but could very well be documented as a new feature.